documentation/MeetingMinutes/2024/MOM-2024-01-31.md

```diff
@@ -0,0 +1,169 @@
+  - Issue #99 "Clarify consent management functionality included in CAMARA-API-access-and-user-consent.md"
+  - Issue #100 "Claims to be used in oAuth Access tokens".
+  - Issue #103 "Remove duplicate documentation after R0.1"
+  - Issue #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern"
+  - Issue #108 "Question to documentation update for security schemes"
+
+- 1 ongoing PRs 
+  
+  - PR #105 "Update CAMARA-API-access-and-user-consent.md login_hint example". Fixes issue #104.
+
+- No new issues raised since last call 17/01
+
+  - N/A
+
+- 3 new PRs since last call 17/01
+
+  - PR #113 "Add CAMARA OIDC Profile". Fixes issues #78 and #90.
+  - PR #114 "Add missing response errors in CIBA flow from GSMA doc". Partially fixes issue #82.
+  - PR #115 "Delete documentation/CAMARA-AuthN-AuthZ-Concept.md". Fixes issue #103.
+
+<br>
+
+## New participant assignment to chair the working group.
+
+- It is announced that Axel Nennker (DT) will take over the chair from the next working group meeting. He replaces Jesús Peña (Telefónica).
+- Jesús and Telefónica will continue to actively contribute to the working group.
+- In his new role, Axel is also proposed to replace Rafal Artych in CODEOWNERS as DT representative.
+- Axel points out that it's not a chairman's role, but a moderator's role, as he will have the same opinion as everyone else. Just like Jesús did in the past. 
+
+## CAMARA/OGW OIDC Profile.
+
+- **Related issues**: #78 "clarify the "credentials" notion in CIBA workflow authentication", #90 "OIDC guidance (what needs to be adhered to?)" & #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern".
+- First contribution sent by Telefónica in PR #113, there are many comments in the proposal. Each comment cannot be reviewed in the call, so Jesús proposes to discuss the two main lines of discussion:
+1. Format of the profile: Delta (only difference from standards, otherwise reference) vs. self-contained (include all relevant information).
+2. Scope of the profile: Suggestion from TEF is to have a first proposal showing the short term solution and include that in v0.2 as per [pull/113#discussion_r1471994271](https://github.com/camaraproject/IdentityAndConsentManagement/pull/113#discussion_r1471994271). Release v0.3 will include further improvements in areas not yet covered.
+
+- Shilpa (DT) points out that purpose short-term solution is already documented in CAMARA-API-access-and-user-consent.md and if some topics are not covered in the profile, then we will not benefit from the existing document that we have in v0.1. Jesús clarifies that the profile contains relevant information that is not part of CAMARA-API-access-and-user-consent.md that is being requested by other CAMARA participants for their current implementations such as the client authtentication, CIBA login_hint, error scenarios, etc.
```

Comment:
- Shilpa (DT) points out that the CAMARA-API-access-and-user-consent.md under release 0.1 already documents the short term agreement. The urgent open issues that need immediate clarifications, are marked (labelled) as r0.2 under issues.  The outcome of those clarifications can be added as an extension to the existing CAMARA-API-access-and-user-consent.md and the release0.2 can be created. She does not see any benefit in creating an OIDC profile which will repeat the short term agreement. The OIDC profile should include the improvements and should get rid of the short term agreement.

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/118/commits/ffc2571d079773b87d1f37f1a389f846982d5f8b